CMakeLists.txt

```diff
@@ -136,6 +138,10 @@ else()
 	endif()
 endif()
 
+if(CMAKE_CXX_STANDARD_DEFAULT EQUAL 98)
+  target_compile_features(libninja PUBLIC cxx_std_11)
+endif()
```

Comment:
something seems wrong about that - isn't `CMAKE_CXX_STANDARD_DEFAULT` just what the compiler will use if not given a specific option-setting?

What if the user explicitly set the standard version to something higher when invoking this CMake?

So I asked on the CMake Slack channel, and @mathstuf said to just use:
```
target_compile_features(libninja PUBLIC cxx_std_11)
```
and CMake will figure it out.